### ISSUE TITLE
Missing Kibana OpenDistro version 1.13.3 Docker image

### ISSUE THREAD

**Issue by czqrny** [opendistro-for-elasticsearch/opendistro-build#793]
**Query**
What question do you have?

Why I can not pull `amazon/opendistro-for-elasticsearch-kibana:1.13.3` Docker image?

**Expected Behavior**
Please describe the behavior you are expecting

This image should be present in Docker registry as `amazon/opendistro-for-elasticsearch:1.13.3` is present.

**Current Behavior**
What is the current behavior?

Kibana opendistro in version 1.13.3 image is missing:

```console
> docker pull amazon/opendistro-for-elasticsearch-kibana:1.13.3
Error response from daemon: manifest for amazon/opendistro-for-elasticsearch-kibana:1.13.3 not found: manifest unknown: manifest unknown
**Failure Information (for bugs)**
Please help provide information about the failure if this is a bug by opening a bug report issue
```

Elasticsearch OpenDistro image in this version exists:

```console
> docker pull amazon/opendistro-for-elasticsearch:1.13.3
1.13.3: Pulling from amazon/opendistro-for-elasticsearch
2d473b07cdd5: Pull complete 
9c08150282af: Pull complete 
9563d7d6b5aa: Pull complete 
d9f19c6631c9: Pull complete 
19b6ddff5271: Pull complete 
af2a9e1a8594: Pull complete 
32c513a03574: Pull complete 
922b31186d67: Pull complete 
5b2900a91e87: Pull complete 
c861bd833319: Pull complete 
10c13cb86476: Pull complete 
1ca5746888f7: Pull complete 
9eb1b9166d6e: Pull complete 
Digest: sha256:2acfa1dcc5f8a2e5aac09bc64c4047c6674755f9b2f57de2a945be182888ea33
Status: Downloaded newer image for amazon/opendistro-for-elasticsearch:1.13.3
docker.io/amazon/opendistro-for-elasticsearch:1.13.3
```

**Relevant information**
Please include any relevant log snippets or files here.

**Comment by pc-jedi** [comment]
The thing is that 1.13.3 is just a patch release for the log4shell CVE.
I guess since Kibana was never effected, since no Java is been used, there is also no patch version of it.

**Comment by stockholmux** [comment]
Yep. @pc-jedi is correct. There is no `amazon/opendistro-for-elasticsearch-kibana:1.13.3` because Kibana didn't change (1.13.3 only fixes the log4j stuff in the ES distribution, Kibana is written in Javascript so it isn't affected)

It sounds like this might be confusing to folks - I'd love to hear feedback to make it more clear.

**Comment by rlevitsky** [comment]
I have the same issue.
Is there any chance that Kibana version will be consistent?

**Comment by stockholmux** [comment]
@rlevitsky I don't think there is a plan to increment the Kibana version since nothing changed.

**Comment by Cai-Chen** [comment]
> @rlevitsky I don't think there is a plan to increment the Kibana version since nothing changed.

But the doc has updated Kibana to 1.13.3 which caused an `Access Denied` error.

**Comment by alejandrohdo** [comment]
in docker-compose.yml replace kibana version with `amazon/opendistro-for-elasticsearch:latest`

**Comment by akshanshshrivastava** [comment]
@alejandrohdo I did the same but got the same error

<img width="564" alt="Screenshot 2022-07-19 at 20 42 55" src="https://user-images.githubusercontent.com/21341620/179785590-4a31aab0-f854-4c89-abf1-bf5d5e6b7205.png">
